Sources/Networking/Operations/PostAttributionDataOperation.swift

```diff
@@ -60,6 +60,9 @@ class PostAttributionDataOperation: NetworkOperation {
 
 }
 
+// Restating inherited @unchecked Sendable from Foundation's Operation
+extension PostAttributionDataOperation: @unchecked Sendable {}
```

Comment:
This will make you shoot yourself in the foot. These operations aren't actually Sendable?